### PR TITLE
get network settings from config.openshift.io

### PR DESCRIPTION
**Pull request by sanchezl**
Pickup new versions of `GetClusterCIDRs` & `GetServiceCIDR` from `github.com/openshift/library-go/pkg/operator/configobserver/network`.

### CODE REVIEW

**Comment by deads2k** [comment]
@sanchezl bump commits may only include files vendor and glide.lock, nothing else

Please reorganize this pull into separate commits and you'll need to re-open your pull to library-go

**Comment by sanchezl** [comment]
Removed fakebump.

File: pkg/operator/configobservation/network/observe_network_test.go [hunk 1]
```diff
@@ -1,33 +1,31 @@
 package network
 
 import (
+	configv1 "github.com/openshift/api/config/v1"
```
Comment (deads2k) [inline pkg/operator/configobservation/network/observe_network_test.go]: nit order

Comment (sanchezl) [reply pkg/operator/configobservation/network/observe_network_test.go]: fixed

**Comment by deads2k** [comment]
green.  and lgtm'd  CI is confused.

**Pull request merged**